client/go.mod

```diff
@@ -0,0 +1,9 @@
+module github.com/quinqu/echo-server/client
```

Comment:
Typically if `go.mod` is checked in, `go.sum` should be as well.

Reply:
I just left it out because go.mod already provides enough information for reproducible builds.

Reply:
I'll add it to the repo though

Reply:
> I just left it out because go.mod already provides enough information for reproducible builds.

This isn't de jure true, though in practice it is. Here's why:

> The go command uses the go.sum file to ensure that future downloads of these modules retrieve the same bits as the first download, to ensure the modules your project depends on do not change unexpectedly, whether for malicious, accidental, or other reasons. Both go.mod and go.sum should be checked into version control.

https://blog.golang.org/using-go-modules

If a remote module were to change its tag (e.g. kingpin pushes a new v2.2.6 pointing to a different commit) it could break your build without go.sum.